app/assert.go

```diff
@@ -0,0 +1,32 @@
+package althea
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/althea-net/althea-chain/config"
+)
+
```

Comment:
This ensures that the node fails to produce a block if the configuration is incorrect, but we have a separate task to make sure the genesis verification fails before this.